main.js

```diff
@@ -90,6 +90,54 @@ const renderMainWindow = () => {
         });
     }
   })();
+  
+  function getDatаNow() {
```

Comment:
@opjs105rogue  rename this function to "getCurrentDatetime()", because data refers to information and not to date.